components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.common/src/main/java/org/wso2/carbon/identity/api/server/application/management/common/ApplicationManagementConstants.java

```diff
@@ -70,6 +70,8 @@ private ApplicationManagementConstants() {
     public static final String FAPI_ALLOWED_CLIENT_AUTHENTICATION_METHODS = "OAuth.OpenIDConnect.FAPI." +
             "AllowedClientAuthenticationMethods.AllowedClientAuthenticationMethod";
     public static final String RSA1_5 = "RSA1_5";
+    public static final String PROP_LOGOUT_RETURN_URL = "logoutReturnURL";
```

Comment:
this can be a URL or Regex pattern

Reply:
In the original SOAP API implementation[1] this is used as logoutReturnUrl. This is kept the same to be consistent among the APIs. 

https://github.com/wso2/carbon-identity-framework/pull/3267/files